.github/workflows/staging-test-tar-x64.yml

```diff
@@ -0,0 +1,679 @@
+name: Process TAR Artifacts X64
```

Comment:
Please change the name to "Test x64 TAR Artifacts"

Reply:
Resolved.